src/Modules/Microsoft.Azure.Functions.PowerShellWorker/Microsoft.Azure.Functions.PowerShellWorker.psm1

```diff
@@ -9,20 +9,25 @@ function CheckIfDurableFunctionsEnabled {
 	}
 }
 
+function GetOrchestrationClientFromModulePrivateData {
+    $PrivateData = $PSCmdlet.MyInvocation.MyCommand.Module.PrivateData
+    $PrivateData['OrchestrationClient']
```

Comment:
We should check that ```$PrivateData``` is not null or empty.
```
if ($PrivateData)
{
   $PrivateData['OrchestrationClient']
}
```

Reply:
Thank you, done.